releases/release-7.1.0.md

```diff
@@ -110,11 +136,25 @@ In v7.1.0, the key new features and improvements are as follows:
 
     For more information, see [documentation](/br/br-checkpoint-restore.md).
 
-* Optimize the strategy of loading statistics [#42160](https://github.com/pingcap/tidb/issues/42160) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes)
+* Optimize the strategy of loading statistics [#42160](https://github.com/pingcap/tidb/issues/42160) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes) **tw:hfxsd**
+
+    Enabling synchronous loading of statistics can significantly reduce the number of statistics that must be loaded during startup, thus improving the speed of loading statistics. This feature increases the stability of TiDB in complex runtime environments and reduces the impact on the overall service when TiDB nodes restart. You can set the parameter [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) to `true` to enable this feature. Note that this parameter is experimental.
+
+    TiDB v7.1.0 also introduces the configuration parameter [`force-init-stats`](/tidb-configuration-file.md#force-init-stats-new-in-v710). This parameter forces TiDB to provide service to the public after the statistics is loaded, avoiding SQL performance issues caused by unloaded statistics during the startup phase. This parameter is disabled by default.
+
+    For more information, see [documentation](/tidb-configuration-file.md).
```

Comment:
```suggestion
   TiDB v7.1.0 introduces lightweight statistics initialization as an experimental feature. Lightweight statistics initialization can significantly reduce the number of statistics that must be loaded during startup, thus improving the speed of loading statistics. This feature increases the stability of TiDB in complex runtime environments and reduces the impact on the overall service when TiDB nodes restart. You can set the parameter [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) to `true` to enable this feature. Note that this parameter is experimental.

    To avoid SQL performance issues caused by unloaded statistics during the startup phase, TiDB v7.1.0 also introduces the configuration parameter [`force-init-stats`](/tidb-configuration-file.md#force-init-stats-new-in-v710). You can control whether to force TiDB to provide service to the public after the statistics is loaded. This parameter is disabled by default.

    For more information, see [documentation](/statistics.md#load-statistics).
```